internal/serve/httphandler/profile_handler.go

```diff
@@ -383,6 +387,10 @@ func (h ProfileHandler) GetOrganizationInfo(rw http.ResponseWriter, req *http.Re
 		resp["payment_cancellation_period_days"] = *org.PaymentCancellationPeriodDays
 	}
 
+	if org.PrivacyPolicyLink != nil {
+		resp["privacy_policy_link"] = org.PrivacyPolicyLink
```

Comment:
You should add the `*` otherwise you are passing the memory address to the map.

```suggestion
		resp["privacy_policy_link"] = *org.PrivacyPolicyLink
```